js/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
This file should be in root folder
Because it is not a part of application scripts